src/main/resources/liquibase/changelog.yml

```diff
@@ -0,0 +1,515 @@
+databaseChangeLog:
```

Comment:
As we discussed please remove this and use .xml only

Reply:
Ok